lib/internal/cluster/master.js

```diff
@@ -67,19 +68,15 @@ cluster.setupMaster = function(options) {
 
   initialized = true;
   schedulingPolicy = cluster.schedulingPolicy;  // Freeze policy.
-  assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
-         `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
-
-  const hasDebugArg = process.execArgv.some((argv) => {
-    return /^(--debug|--debug-brk)(=\d+)?$/.test(argv);
-  });
+  if (schedulingPolicy !== SCHED_NONE && schedulingPolicy !== SCHED_RR)
```

Comment:
> Are you saying that inlining conditionals and avoiding function calls is a big perf boost? 

I'm not sure if the difference in perf is from avoiding the overhead of a function call to `assert()` or from avoiding the need to create the template string that gets passed to `assert()` or both of those things or...